curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md

```diff
@@ -9,7 +9,12 @@ dashedName: step-81
 
 When the user has a calorie deficit, the `remainingCalories` value will be negative. You don't want to display a negative number in the result string.
 
-`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number. In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
+`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number. 
```

Comment:
It looks like there was a misunderstanding on how the code example should be added.
As mentioned earlier, you shouldn't delete the instructions for the challenge.
The goal here is to just add a code example for `Math.abs`.

Here should be the correct complete description.
Please copy this and make those changes 👍 

````md
# --description--

When the user has a calorie deficit, the `remainingCalories` value will be negative. You don't want to display a negative number in the result string.

`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number.

```js
const num = -5;
Math.abs(num); // 5
```

In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
````